components/director/hack/gqlgen.go

```diff
@@ -17,7 +18,7 @@ func main() {
 	}
 
 	err = api.Generate(cfg,
-		api.AddPlugin(scopesdecorator.NewPlugin("schema.graphql")))
+		api.AddPlugin(scopesdecorator.NewScopesDecoratorPlugin("schema.graphql")), api.AddPlugin(descriptionsdecorator.NewSDescriptionsDecoratorPlugin("schema.graphql")))
```

Comment:
The package name contains this information so there is no need to put it also into the func name.

```suggestion
		api.AddPlugin(scopesdecorator.NewPlugin("schema.graphql")), api.AddPlugin(descriptionsdecorator.NewPlugin("schema.graphql")))
```